concent_api/core/views.py

```diff
@@ -617,6 +619,16 @@ def deserialize_message(raw_message_data):
         raise Http400("Unable to deserialize Golem Message: {}.".format(exception))
 
 
+def validate_report_computed_task_time_window(report_computed_task):
+    if not isinstance(report_computed_task, message.ReportComputedTask):
+        raise Http400("Expected ReportComputedTask")
+
+    task_to_compute_younger_then_report_computed = report_computed_task.task_to_compute.timestamp > report_computed_task.timestamp
```

Comment:
please either rename it to boolean format name (is*, has*,) -> `is_task_to_compute_younger_then_report_computed` or just inline variable